packages/slice-machine/tests/server/apiController/simulator.test.ts

```diff
@@ -57,130 +55,5 @@ describe("simulator controller", () => {
       requestWithPreviewUrl
     );
     expect(previewCheckResponse.manifest).toBe("ok");
-    expect(previewCheckResponse.dependencies).toBe("ko");
-  });
-
-  test("it should return ok when all the legacy next required deps are installed", async () => {
```

Comment:
do we no longer check that the user has installed all the deps?

Reply:
nope!